mu4/appshell/qml/DevTools/NotationDialogs/MU3Dialogs.qml

```diff
@@ -5,13 +5,30 @@ import MuseScore.Ui 1.0
 Item {
     id: root
 
-    FlatButton {
-        width: 140
+    Column {
+        anchors.fill: parent
+        anchors.margins: 12
 
-        text: qsTrc("devtools", "Measure properties")
+        spacing: 12
 
-        onClicked: {
-            api.launcher.open("musescore://notation/measureproperties?index=0")
+        FlatButton {
+            width: 140
+
+            text: qsTrc("devtools", "Measure properties")
```

Comment:
I think, no need to make translatable strings for devtools

Reply:
fixed